components/seldon-request-logger/Makefile

```diff
@@ -7,8 +7,8 @@ CLIENT_ID=sd-api
 OIDC_USERNAME=admin@seldon.io
 OIDC_PASSWORD=xxxxxx
 OIDC_SCOPES=openid profile email groups
-DEPLOY_API_HOST=http://xx.xx.xx.xx/seldon-deploy/api/v1alpha1
-OIDC_PROVIDER=http://xx.xx.xx.xx/auth/realms/deploy-realm
+#DEPLOY_API_HOST=http://xx.xx.xx.xx/seldon-deploy/api/v1alpha1
```

Comment:
Seems comment left out

Reply:
I commented out so as not to assume deploy exists when running locally but should really improve Makefile for the 2 options of running with and without deploy